### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by EVAST9919**
1. Hide options when entering the game
2. Hide letterbox position sliders while non-use

### CODE REVIEW

File: osu.Game/OsuGame.cs [hunk 1]
```diff
@@ -242,16 +242,19 @@ private bool globalHotkeyPressed(InputState state, KeyDownEventArgs args)
                     return true;
             }
 
-            if (state.Keyboard.ControlPressed)
+            if (currentScreen.UseSettingsOverlays)
```
Comment (peppy) [inline osu.Game/OsuGame.cs]: i don't think we want to do this. options are intended to be accessible at any point game-wide, including during gameplay if you wish.

Comment (EVAST9919) [inline osu.Game/OsuGame.cs]: How about only on pause?

Comment (peppy) [inline osu.Game/OsuGame.cs]: i'd rather not introduce arbitrary limits until we find a valid reason to do so.

File: osu.Game/Screens/OsuGameScreen.cs [hunk 2]
```diff
@@ -22,6 +22,8 @@ public abstract class OsuScreen : Screen
 
         internal virtual bool ShowOverlays => true;
 
+        internal virtual bool UseSettingsOverlays => true;
```
Comment (peppy) [inline osu.Game/Screens/OsuGameScreen.cs]: why introduce a special case for settings? is there a reason for not making it part of `ShowOverlays` above?

**Pull request merged**